server/cmd/mmctl/client/client.go

```diff
@@ -44,7 +44,9 @@ type Client interface {
 	CreatePost(ctx context.Context, post *model.Post) (*model.Post, *model.Response, error)
 	GetPostsForChannel(ctx context.Context, channelID string, page, perPage int, etag string, collapsedThreads bool, includeDeleted bool) (*model.PostList, *model.Response, error)
 	GetPostsSince(ctx context.Context, channelID string, since int64, collapsedThreads bool) (*model.PostList, *model.Response, error)
+	DeletePost(ctx context.Context, postId string) (*model.Response, error)
 	DoAPIPost(ctx context.Context, url string, data string) (*http.Response, error)
+	DoAPIDelete(ctx context.Context, url string) (*http.Response, error)
```

Comment:
Do we need this?

Reply:
Should i remove this. Then i will have to change DoAPIDelete in post_test to DeletePost. I think we dont need it. What do you say.